workshop/exercise-5/README.md

```diff
@@ -55,4 +55,18 @@ The file should look something like:
 	kubectl apply --filename forward-ingress.yaml
 	```
 
-Continue on to [exercise 6](../exercise-6/README.md).
\ No newline at end of file
+### Try it again
+
+Now that we've setup our DNS routing, let's try our `curl` command again
+using the DNS hostname:
+
+```
+curl -X POST -H 'Content-Type: application/json' fib-knative.default.bmv-knative.us-east.containers.appdomain.cloud/fib -d '{"number":5}'
```

Comment:
did you test this change? I'm unsure if the service.yaml needs to be re-deployed.

Reply:
yup - knative will dynamically update things - pretty cool